internal/scorecard/alpha/bundle.go

```diff
@@ -23,15 +23,15 @@ import (
 )
 
 // getBundleData tars up the contents of a bundle from a path, and returns that tar file in []byte
-func (o Scorecard) getBundleData() (bundleData []byte, err error) {
+func (o Scorecard) GetBundleData() (bundleData []byte, err error) {
```

Comment:
It is 0% covered by tests. See; https://coveralls.io/builds/30552210/source?filename=internal/scorecard/alpha/bundle.go
wdyt about add a test to check this one?

Reply:
thanks, added bundle_test.go unit test.